contrib/metrics/README.md

```diff
@@ -0,0 +1,76 @@
+# zcashd metrics
+
+## Metrics UI
+
+This is the user interface that `zcashd` displays by default when run. It
+displays a small selection of interesting metrics, but is not intended for
+programmatic consumption.
+
+## RPC methods
+
+`zcashd` provides the following JSON-RPC methods that expose node metrics:
+
+- Chain:
+  - `getblockchaininfo`: Various state info regarding block chain processing.
+  - `gettxoutsetinfo`: Statistics about the unspent transparent transaction output set.
+  - `getmempoolinfo`: Details on the active state of the TX memory pool.
+- P2P network:
+  - `getnetworkinfo`: Various state info regarding P2P networking.
+  - `getpeerinfo`: Data about each connected network node.
+  - `getdeprecationinfo`: The current node version and deprecation block height.
+- Miscellaneous
+  - `getmemoryinfo`: Information about memory usage.
+  - `getmininginfo`: Mining-related information.
+  - `getinfo` (deprecated): A small subset of the above metrics.
+
+You can see what each method provides with `zcash-cli help METHOD_NAME`.
+
+## Prometheus support
+
+`zcashd` can optionally expose an HTTP server that acts as a Prometheus scrape
+endpoint. The server will respond to `GET` requests on any request path.
+
```

Comment:
```suggestion

Note that HTTPS is not supported, and therefore connections to the endpoint are
not encrypted or authenticated. Access to the endpoint should be assumed to
compromise the privacy of node operations, by the provided metrics and/or by
timing side channels. Enabling the endpoint is **strongly discouraged** if the
node has a wallet holding live funds.

```

Reply:
Also add this to `doc/security-warnings.md`.

Reply:
Someone else will have to add it to `security-warnings.md` as that's now over in the ReadTheDocs Gitlab repo, and I can't log in there at the moment.